openmc/material.py

```diff
@@ -908,22 +899,51 @@ def get_nuclide_atom_densities(self):
 
         return nuclides
 
-    def get_nuclide_activity(self):
-        """Return activity in [Bq] for each nuclide in the material
+    def get_activity(self, normalization: str = 'total', by_nuclide: bool = False):
```

Comment:
Can we make the default normalization `'volume'`? since that's the case that would get returned by default from just applying decay constants to the result of `get_nuclide_atom_densities()` (i.e. no multiplier necessary and no need for `volume` to be set. @pshriwise would you agree?

Reply:
Defaulting to ```volume``` sounds good to me.
It would just need a few changes to the tests that I've added as suggestions below.
Let us know if that is ok with you @pshriwise

Reply:
Yep! I think that's the right way to go. 💯